Formula/ctop.rb

```diff
@@ -15,14 +15,9 @@ class Ctop < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    src = buildpath/"src/github.com/bcicen/ctop"
-    src.install buildpath.children
-    src.cd do
-      system "make", "build"
-      bin.install "ctop"
-      prefix.install_metafiles
-    end
+    system "make", "build"
+    bin.install "ctop"
+    prefix.install_metafiles
```

Comment:
Not necessary.

https://github.com/Homebrew/brew/blob/a95dd7d48588099fae744cf8970c9e7ea25368bd/Library/Homebrew/build.rb#L159